target/communication/fi_ibex_commands.py

```diff
@@ -54,6 +54,60 @@ def ibex_char_mem_op_loop(self) -> None:
         time.sleep(0.01)
         self.target.write(json.dumps("CharMemOpLoop").encode("ascii"))
 
+    def ibex_char_flash_read(self) -> None:
+        """ Starts the ibex.char.flash_read test.
+        """
+        # IbexFi command.
+        self._ujson_ibex_fi_cmd()
+        # CharFlashRead command.
+        time.sleep(0.01)
+        self.target.write(json.dumps("CharFlashRead").encode("ascii"))
```

Comment:
nit: I guess something like "IbexCharFlashRead" would help to prevent potential confusion (down the line but not really worth the effort changing the encoding :)

Reply:
Thanks for your comment! I think this might be redundant as you have to send these two commands to the code to get the test started:
```
IbexFi
CharFlashRead
```

Reply:
thanks, yes